rest-service/manager_rest/test/endpoints/test_nodes.py

```diff
@@ -427,14 +427,24 @@ def test_node_and_node_instance_properties(self):
 
 
 class NodesCreateTest(base_test.BaseServerTestCase):
-    def test_create_single(self):
```

Comment:
Do we not want to test creating a single node as well?

Reply:
there isn't a way to create a single node, this only implements the "create batch" endpoint, which always takes a list. A test for 1-item list isn't meaningfully different I don't think?